### PR TITLE
refactor: Restructured the database structure

### PR DESCRIPTION
**Pull request by liu-Jinke**
### Restructured the database structure

1.  Added physical structure model to facilitate better understanding of database structure

2.  Change the database schema

3.  Add folder 2020-07-10 to record our work today

### CODE REVIEW

**Review by KONY128 — APPROVED** [review]
老铁没毛病

**Review by KONY128 — CHANGES_REQUESTED** [review]
你的文档结构还有图片等等都有问题，请按照我的规范例子来进行文档书写

**Review by sdjn-wxy — CHANGES_REQUESTED** [review]
请按照文档结构的标准命名方式进行一下修改

File: 2020-07-10/数据库设计问题.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# 数据库设计问题
+
+1、为课程添加类型标识，如必修课，通选课，任选课
```
Comment (KONY128) [inline 2020-07-10/数据库设计问题.md]: use h3

File: 2020-07-10/数据库设计问题.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (KONY128) [inline 2020-07-10/数据库设计问题.md]: If it's convenient  for u, please add your pic into .md files

**Review by KONY128 — CHANGES_REQUESTED** [review]
就按照我要求的来改吧

**Review by KONY128 — APPROVED** [review]
I think this is okay. : )

Good job.

**Pull request merged**